src/operator/merge.ts

```diff
@@ -149,7 +148,7 @@ export function mergeStatic<T, R>(...observables: Array<ObservableInput<any> | S
     concurrent = <number>observables.pop();
   }
 
-  if (observables.length === 1) {
+  if (scheduler === null && observables.length === 1) {
```

Comment:
oh weird, that must have slipped in when I branched from master. yes, it's a bug fix that I was tracking down earlier today but didn't get to finish writing a test for. I can try to remove it from this change set, or alternatively add the test I was going to write and push again :[